python/adelphi/setup.py

```diff
@@ -1,5 +1,16 @@
 from setuptools import setup
 
+PY3 = sys.version_info[0] == 3
+
+dependencies = [
+    'cassandra-driver >= 3.24.0',
+    'click >= 7.1.2',
+    'PyGithub >= 1.45'
+    ]
+
+if not PY3:
+    dependencies.append('backports.tempfile')
+
```

Comment:
Add the backport of py3k tempfile functionality for Python 2.7 support.  Logic here borrowed from what the Python driver does.